.kokoro/presubmit/spanner_perf_bench.cfg

```diff
@@ -0,0 +1,38 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Build logs will be here
+action {
+  define_artifacts {
+    regex: "**/*sponge_log.xml"
+  }
+}
+
+# Download resources for system tests (service account key, etc.)
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-ruby"
+
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Use the trampoline script to run in docker.
+build_file: "google-cloud-ruby/.kokoro/trampoline_v2.sh"
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/yoshi-ruby/release"
+}
+
+env_vars: {
+  key: "TRAMPOLINE_BUILD_FILE"
+  value: ".kokoro/integration.sh"
```

Comment:
Which test do you want this job to run? Currently, it is triggering the integration.sh script without specifying a test to run (which probably results in a noop).

Reply:
I would like this to run benchmarking tests located in Google3, the build file of which is being triggered by a .cfg file in Google3

Reply:
I see. When I look at the corresponding .cfg file in google3 (which it looks like you already added in internal cl/463615654), it looks like it expects this file to be located not here but in google3 also. (See the `build_config_dir` field in the file you added there: it points to a location in google3 rather than here.) And in fact it looks like that file is already present in google3. So I suspect there's actually nothing to be done here, and this pull request is not necessary.

If you're confused, feel free to ping me on internal chat. My internal ldap is "dazuma".

Reply:
And In fact, yes, I can confirm that the test is already running on every pull request. If you look at, for example, the checks in https://github.com/googleapis/google-cloud-ruby/pull/18914, you'll see a failing test related to benchmarking; that's your test.

This of course brings up additional issues:

1. The fact that the test is _failing_. On a first glance, it looks like this is resource/quota related.
2. The fact that the benchmarking test is running for pull requests that have nothing to do with spanner. This repository hosts well over 100 libraries, most of which are not spanner related, and we can't have this spanner benchmarking test as part of their presubmit. It would be too much of a strain on your resources to have benchmarking tests running potentially hundreds of times a day on completely unrelated pull requests, and it's not acceptable for your benchmarking test to block pull requests not related to spanner. (We could potentially deal with this by having the test runner check whether or not the pull request actually modified the spanner client. Our existing integration tests do this already, and I can provide sample code for doing the check.)

I'll open an internal issue to deal with this. Also cc-ing @bajajneha27 who worked on the original benchmarking test and may be able to help with why it's failing.